CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to ParquetSharp.DataFrame
+
+Thank you for your interest in contributing! 🎉
+
+This library is primarily built and maintained to serve G-Research’s needs. While we welcome contributions, our main focus is ensuring the project continues to meet our requirements. We are more likely to accept bug fixes and documentation updates but may be conservative about new features.
+
+We will review issues and pull requests on a **best-effort** basis.
+
+## Issues
+
+Please report bugs and feature requests by opening an [Issue](https://github.com/G-Research/ParquetSharp.DataFrame/issues). If possible, check for existing issues before submitting a new one.
+
+When filing a bug report, follow the provided template, which asks for the following information:
```

Comment:
No need to duplicate the issue form contents, linking to the new issue form is sufficient